trading_ig/rest.py

```diff
@@ -799,6 +799,9 @@ def update_working_order(
         session=None,
     ):
         """Updates an OTC working order"""
+        VERSION = 2
+        if good_till_date is not None and type(good_till_date) is not int:
+            good_till_date = conv_datetime(good_till_date, VERSION)
```

Comment:
This is just copied from create_working_order(): https://github.com/ig-python/ig-markets-api-python-library/blob/58e13db2e808b01fd5025cffbc4204b4ffe3bd19/trading_ig/rest.py#L734-L736